go-controller/pkg/crd/egressservice/v1/apis/informers/externalversions/factory.go

```diff
@@ -80,14 +79,6 @@ func WithNamespace(namespace string) SharedInformerOption {
 	}
 }
 
-// WithTransform sets a transform on all informers.
-func WithTransform(transform cache.TransformFunc) SharedInformerOption {
-	return func(factory *sharedInformerFactory) *sharedInformerFactory {
-		factory.transform = transform
-		return factory
-	}
-}
-
```

Comment:
I am not sure why this happened, if I run update-codegen locally it is added back, can you re-run the script and check if it will be added back?

Reply:
oh! yes, i was using outdated `informer-gen` binary, its corrected now.